examples/kops-kube-vpc/Makefile

```diff
@@ -0,0 +1,80 @@
+# Hardcoding value of 3 minutes when we check if the plan file is stale
+STALE_PLAN_FILE := `find "tf.out" -mmin -3 | grep -q tf.out`
+SHELL := /bin/bash
+
+AWS_AVAILABILITY_ZONE := `terraform output -json | jq '.availability_zone.value' | tr -d '"'`
+KOPS_STATE_BUCKET := `terraform output -json | jq '.state_bucket_id.value' | tr -d '"'`
+KOPS_CLUSTER_NAME := `terraform output -json | jq '.subdomain_zone_name.value' | tr -d '"'`
+KOPS_CLUSTER_ZONE := `terraform output -json | jq '.subdomain_zone_id.value' | tr -d '"'`
+KOPS_AWS_VPC := `terraform output -json | jq '.vpc_id.value' | tr -d '"'`
+
+## Check if tf.out is stale (Older than 2 minutes)
+check-plan-file:
+	@if ! ${STALE_PLAN_FILE} ; then \
+		echo "ERROR: Stale tf.out plan file (older than 3 minutes)!"; \
+		exit 1; \
+	fi
+
+## Runs terraform get and terraform init for env
+init:
+	@terraform get
+	@terraform init
+
+## tergaform plan (makes everything)
+plan:
+	@terraform plan -out=tf.out
+
+## terraform apply
+apply: check-plan-file
+	@terraform apply tf.out
+
+## Prepare the environment with a subdomain and vpc for kops
+prepare-environment: plan apply
+
+## Generate terraform plan using `kops`
+generate-kops-plan:
```

Comment:
I'd like if we could follow the same naming that we have in the existing Makefile in our repo. For example `make kops-create-cluster`, `make kops-update-cluster` etc. We should strive to be as consistent as possible to when we switch projects we can still rely on "muscle memory".